recipe/meta.yaml

```diff
@@ -268,6 +270,7 @@ about:
   dev_url: https://docs.python.org/devguide/
 
 extra:
+  feedstock-name: python
```

Comment:
@xhochy I incorporated https://github.com/conda-forge/python-feedstock/pull/393 here, is that OK?

Reply:
Yes, sure!